catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -597,31 +598,30 @@ private UpdateRequest populateUpdateRequestPolicyMap(UpdateRequest updateRequest
         return updateRequest;
     }
 
-    private UpdateRequest populateMetacards(UpdateRequest updateRequest) throws IngestException {
-        final String attributeName = updateRequest.getAttributeName();
-
+    protected UpdateRequest populateMetacards(UpdateRequest updateRequest) throws IngestException {
```

Comment:
❗️ Making a `private` method package-private or `protected` for the sake of unit testing is usually not a good idea. Since this class is very dense and complex and difficult to unit test, is there a way we could test the change with an itest instead? Or we could manually test the change for now and open a high-priority tech debt ticket to refactor this code and add unit testing. Thoughts?